addons/base_vat/models/res_partner.py

```diff
@@ -363,3 +363,14 @@ def check_vat_al(self, vat):
             return stdnum.al.vat.is_valid(vat)
         except ImportError:
             return True
+
+    def check_vat_nl(self, vat):
```

Comment:
Add a comment as it's done to the Ireland check.
https://github.com/odoo/odoo/blob/11.0/addons/base_vat/models/res_partner.py#L205

Reply:
You can refer this website in your commit : https://business.gov.nl/regulation/using-checking-vat-numbers/